enterprise/api/v1/permissions.py

```diff
@@ -19,9 +19,11 @@ def has_permission(self, request, view):
         """
         Returns True if the requesting user is either 'staff' or belong to specific group, False otherwise.
         """
-        return super(IsAdminUserOrInGroup, self).has_permission(request, view) or request.user.groups.filter(
-            name__in=self.ALLOWED_API_GROUPS
-        ).exists()
+        return (
```

Comment:
Probably worth updating the docstring to indicate the group check for values in the request query_params.permissions